setup.py

```diff
@@ -9,7 +9,7 @@
     long_description=open("README.md").read(),
     long_description_content_type="text/markdown",
     author_email="dataanalysis@diamond.ac.uk",
-    packages=find_packages(),
+    packages=["ParProcCo", "ParProcCo/slurm", "ParProcCo/test"],
```

Comment:
Is this definitely right? The documentation seems to link it should be `["ParProcCo", "ParProcCo.slurm", "ParProcCo.test"]` but I'm not sure what you're trying to change here